facia/app/controllers/FaciaController.scala

```diff
@@ -352,27 +353,55 @@ trait FaciaController
 
   def renderShowMore(path: String, collectionId: String): Action[AnyContent] =
     Action.async { implicit request =>
-      frontJsonFapi.get(path, fullRequestType).flatMap {
-        case Some(pressedPage) =>
-          val containers = Front.fromPressedPage(pressedPage, Edition(request), adFree = request.isAdFree).containers
-          val maybeResponse =
-            for {
-              (container, index) <- containers.zipWithIndex.find(_._1.dataId == collectionId)
-              containerLayout <- container.containerLayout
-            } yield {
-              val remainingCards: Seq[FaciaCardAndIndex] = containerLayout.remainingCards.map(_.withFromShowMore)
-              val adFreeFilteredCards: Seq[FaciaCardAndIndex] = if (request.isAdFree) {
-                remainingCards.filter(c => !checkIfPaid(c.item))
-              } else {
-                remainingCards
+      if (request.forceDCR) {
+        frontJsonFapi.get(path, fullDCRRequestType).flatMap {
```

Comment:
Checking my understanding here: We don't expect this to fail due to a lack of `pressed.v2.full.dcr.json` in the bucket. This new json type will be created from the cron job a while after this code gets merged due to the changes in `facia-press`. Is this correct?

Reply:
I think will will break CODE fronts as I am not sure about the frequency of the job. Here's a current example of a CODE pressed json files in the bucket:

<img width="1239" alt="image" src="https://user-images.githubusercontent.com/19683595/175290886-afb8d80e-e4d2-4a3c-b34b-74f4e9fdcd9c.png">